onnxruntime/core/providers/nv_tensorrt_rtx/nv_execution_provider.cc

```diff
@@ -104,6 +111,112 @@ static bool IsSupportedInputOutputDataType(ONNXTensorElementDataType data_type)
   }
 }
 
+Status NvExecutionProvider::GetExtSemaphore(const struct GraphicsInteropParams* graphicsInteropParams, struct FenceInteropParams* fenceInteropParams, void** extSemFence)
+{
+  if (!info_.has_user_compute_stream) 
```

Comment:
`has_user_compute_stream` is very specific to this EP. how will this work for an EP that doesn't currently support user provided streams? would they create a stream using the ORT API and pass it in somehow (e.g. in the params to one of the new API calls)? we don't really want to manually have to add an option to all the EPs for this sort of thing.